BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -0,0 +1,212 @@
+        backgroundImageView = UIImageView(image: UIImage(named: "mountains"))
+    }
+    
+    private func configureWelcomLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineHeightMultiple = 1.09
+
+        let attributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 16.0)!,
+             .foregroundColor: UIColor(hexNumber: 0x000000),
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        label.attributedText = NSAttributedString(string: "ДОБРО ПОЖАЛОВАТЬ В", attributes: attributes)
+
+        welcomeLabel = label
+    }
+    
+    private func configureAppNameLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineHeightMultiple = 1.09
+
+        let stringAttributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.SFProDisplayBold.rawValue, size: 24.0)!,
+             .foregroundColor: UIColor.black,
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        label.attributedText = NSAttributedString(string: "HABIT HERO", attributes: stringAttributes)
+
+        appNameLabel = label
+    }
+    
+    private func configureLogoImageView() {
+        logoImageView = UIImageView(image: UIImage(named: "logo"))
+        
+        configureLogoLabel()
+        logoImageView.addSubview(logoLabel)
+    }
+    
+    private func configureLogoLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .left
+
+        let attributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.interExtraBold.rawValue, size: 15.0)!,
+             .foregroundColor: UIColor(hexNumber: 0xFFFFFF),
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        label.attributedText = NSAttributedString(string: "ЛОГО", attributes: attributes)
+        
+        logoLabel = label
+    }
+    
+    private func configurePhraseLabel() {
+        let label = UILabel()
+        
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineHeightMultiple = 1.26
+        paragraphStyle.lineBreakMode = .byWordWrapping
+
+        let attributes: [NSAttributedString.Key: Any] = [
+             .kern: -0.41,
+             .font: UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 14.0)!,
+             .foregroundColor: UIColor.black,
+             .paragraphStyle: paragraphStyle
+        ]
+        
+        let phrase =
+"""
+НЕ ЗАПОЛНЯЮТ ПУСТОТУ,  ОНИ ЕЕ СОЗДАЮТ!
+НЕ ЗАПОЛНЯЮТ ПУСТОТУ,  ОНИ ЕЕ СОЗДАЮТ!
+ТУТ КАКАЯ-ТО ФРАЗА
+"""
```

Comment:
Do we plan adding localisation? I suppose we should

Reply:
Localization added